packages/pangea-sdk/pangea/services/embargo.py

```diff
@@ -56,7 +56,7 @@ class Embargo(ServiceBase):
     """Embargo service client.
 
     Provides methods to interact with Pangea Embargo Service:
-        https://docs.dev.pangea.cloud/docs/api/embargo
+        https://pangea.cloud/docs/api/embargo
```

Comment:
should be docs.pangea.cloud (no aws.us here)

Reply:
According to @r3dcrosse it is https://pangea.cloud/docs/xxx